packages/dx-react-chart/src/plugins/zoom-pan.tsx

```diff
@@ -95,49 +93,46 @@ class ZoomAndPanBase extends React.PureComponent<ZoomAndPanProps, ZoomAndPanStat
     };
   }
 
-  handleStart(zoomRegionKey: string, e: any) {
-    this.offset = getOffset(e.currentTarget);
+  handleStart(zoomRegionKey: string, e: any, rootRef: React.RefObject<Element>) {
+    if (!isOver(e, rootRef.current!.getBoundingClientRect())) {
+      return;
+    }
+    this.offset = getOffset(rootRef.current!);
       // Rectangle mode should be canceled if "zoomRegionKey" is released during mouse movevent or
       // not pressed when mouse is up. To do it access to "event" object is required in
-      // "handleMouseMove" and "handleMouseUp".
+      // "handleMove" and "handleEnd".
       // TODO: Provide rectangle mode canceling.
     if (isKeyPressed(e, zoomRegionKey)) {
       this.rectOrigin = [e.pageX - this.offset[0], e.pageY - this.offset[1]];
     }
     if (e.touches && e.touches.length === 2) {
       this.multiTouchDelta = getDeltaForTouches(e.touches).delta;
     }
+    this.lastCoordinates = [e.clientX - this.offset[0], e.clientY - this.offset[1]];
```

Comment:
Somewhere `pageX` and `pageY` are used (here in this plugin and in `EventTracker`) and somewhere - `clientX` and `clientY`.
Can we bring it to single variant? Looks like it should be `pageX` and `pageY`.

Reply:
ok, let's make `pageX` and `pageY`